deebotozmo/cli.py

```diff
@@ -0,0 +1,591 @@
+#!/usr/bin/env python3
+"""Cli module."""
+import asyncio
+import base64
+import configparser
+import itertools
+import json
+import logging
+import mimetypes
+import os
+import platform
+import sys
+import time
+from dataclasses import asdict
+from functools import wraps
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+import aiohttp
+import click
+
+from deebotozmo.commands import (
+    Charge,
+    CleanCustomArea,
+    CleanPause,
+    CleanResume,
+    CleanSpotArea,
+    CleanStart,
+    PlaySound,
+    SetFanSpeed,
+    SetWaterLevel,
+)
+from deebotozmo.ecovacs_api import EcovacsAPI
+from deebotozmo.events import (
+    BatteryEvent,
+    CleanLogEvent,
+    FanSpeedEvent,
+    MapEvent,
+    RoomsEvent,
+    StatsEvent,
+    StatusEvent,
+    WaterInfoEvent,
+)
+from deebotozmo.models import Vacuum, VacuumState
+from deebotozmo.util import md5
+from deebotozmo.vacuum_bot import VacuumBot
+
+
+def coro(func: Callable) -> Callable:
+    """Wrap around a function to run it in coroutine."""
+
+    @wraps(func)
+    def wrapper(*args: Tuple, **kwargs: Dict) -> Any:
+        loop = asyncio.get_event_loop()
+        return loop.run_until_complete(func(*args, **kwargs))
+
+    return wrapper
+
+
+def config_file() -> str:
+    """Return the path to config file."""
+    if platform.system() == "Windows" and "APPDATA" in os.environ:
+        return os.path.join(str(os.getenv("APPDATA")), "deebotozmo.conf")
+    return os.path.expanduser("~/.config/deebotozmo.conf")
+
+
+def config_file_exists() -> bool:
+    """Check if the config file exists."""
+    return os.path.isfile(config_file())
+
+
+def read_config() -> configparser.ConfigParser:
+    """Read and parse the config file."""
+    parser = configparser.ConfigParser()
+    parser.read(config_file())
+    return parser
+
+
+def read_config_old() -> configparser.SectionProxy:
```

Comment:
Please mark that this method is deprecated and should be removed with the next major release